docs/architecture/adr-018-network-upgrades.md

```diff
@@ -0,0 +1,68 @@
+# ADR 018: Network Upgrades
+
+## Status
+
+Proposed
+
+## Changelog
+
+- 2023/03/29: Initial draft
+- 2023/08/29: Modified to reflect the proposed decision and the detailed design
+
+## Context
+
+There are three requirements in Celestia's upgrading mechanism that diverges from how the standard Cosmos SDK currently operates:
+
+- Upholding a stance of minimal on-chain governance and a preference towards off-chain governance or social consensus. Using the option of exit as a tool to ensure that those with explicit power (validators, perhaps) are held accountable with those of implicit power i.e. the wider community of users that benefit from Celestia
```

Comment:
I guess I could rephrase these as goals rather than requirements (i.e. directions we're trying to head in rather than tangible requirements)

> One that acts as a forcing function for node operators to practice coordinating among themselves.

Doesn't signalling automate this coordination aspect.

> We will be well versed in picking our own source of truth

The good this with token voting is that there is a medium with which all delegators can state their preference/decision. With social consensus, it can sometimes be not so clear what _everyones_ preferences are